### PR TITLE
Update data czar skills and remove backup

### PR DESCRIPTION
**Pull request by lamagnifica**
### Reviewers
- [x] Subject matter expert: @stroilova 
- [x] Subject matter expert: @e0d 
- [x] Doc team review (sanity check): @pdesjardins or @catong 
### Testing
- [ ] Ran ./run_tests.sh without warnings or errors
### Post-review
- [n/a] Add description to release notes task as a comment
- [ ] Squash commits

### CODE REVIEW

File: en_us/data/source/internal_data_formats/data_czar.rst [hunk 1]
```diff
@@ -79,6 +73,9 @@ Technical Skills
 - Ability to download large files from Amazon Web Service (AWS) Simple Storage
   Service (S3).
 
+- Ability to set up a big data pipeline and run scripts to download files in
```
Comment (e0d) [inline en_us/data/source/internal_data_formats/data_czar.rst]: a secure, internal data distribution pipeline?

Comment (stroilova) [inline en_us/data/source/internal_data_formats/data_czar.rst]: Agreed. Thanks @e0d .

**Comment by stroilova** [comment]
LGTM thanks :+1:  

FYI @shnayder  on this update as well.

**Comment by lamagnifica** [comment]
@e0d?

**Comment by srpearce** [comment]
:+1:

**Comment by e0d** [comment]
:+1:

**Pull request merged**